src/esssans/normalization.py

```diff
@@ -33,22 +51,33 @@ def solid_angle_of_rectangular_pixels(
     -------
     :
         The solid angle of the detector pixels, as viewed from the sample position.
-        Any masks that have a dimension common to the dimensions of the position
-        coordinate are retained to the output.
+        Any coords and masks that have a dimension common to the dimensions of the
+        position coordinate are retained to the output.
     """
     L2 = scn.L2(data)
     omega = (pixel_width * pixel_height) / (L2 * L2)
-    solid_angle = sc.DataArray(data=omega)
-    omega_dims = set(omega.dims)
-    for key, mask in data.masks.items():
-        if set(mask.dims).issubset(omega_dims):
-            solid_angle.masks[key] = mask
-    return solid_angle
+    dims = set(data.dims) - set(omega.dims)
+    return concepts.rewrap_reduced_data(prototype=data, data=omega, dim=dims)
+
+
+def solid_angle_rectangular_approximation(
```

Comment:
Why is the wrapper needed here if we are simply calling `solid_angle_of_rectangular_pixels`?

Reply:
It is not, I just took what was in `ess.sans`, where for some reason the coords are extracted outside the function and passed alongside the data. I do not know why that decision was made?

Reply:
I removed the wrapper now (and updated the tests)